client_end_user/shared/src/commonMain/kotlin/presentation/SearchTaxi/SearchTaxiUIState.kt

```diff
@@ -0,0 +1,13 @@
+package presentation.SearchTaxi
+
+data class SearchTaxiUIState(
+    val searchTerm: String = "",
+    val isSearchButtonEnabled: Boolean = false,
+    val result : List<SearchResultUIState>   = emptyList(),
+)
+
+data class SearchResultUIState(
+    val country: String = "",
+    val city: String = "",
+    val address: String = "",
+)
```

Comment:
if `SearchResultUIState` was used only in `SearchTaxiUIState` data class then add it inside it else seprate it in another file
or if it was used in this directory only add internal before it